src/inngest/functions/index-lessons.ts

```diff
@@ -2,13 +2,15 @@ import {inngest} from '@/inngest/inngest.server'
 import {INDEX_LESSONS_FOREVER} from '@/inngest/events/index-lessons-forever-event'
 import {google} from 'googleapis'
 
-const base64EncodedServiceAccount = process.env
-  .GOOGLE_SERVICE_ACCOUNT_ENCODED as string
+const base64EncodedServiceAccount =
+  process.env.GOOGLE_SERVICE_ACCOUNT_ENCODED ?? ('' as string)
 const decodedServiceAccount = Buffer.from(
   base64EncodedServiceAccount,
   'base64',
 ).toString('utf-8')
-const credentials = JSON.parse(decodedServiceAccount)
+const credentials = decodedServiceAccount
+  ? JSON.parse(decodedServiceAccount)
+  : ''
```

Comment:
build was throwing type errors in dev because I don't have t`GOOGLE_SERVICE_ACCOUNT_ENCODED` defined so this makes it not break